### ISSUE TITLE
Presenter can drag presentation off screen

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#1682]
Originally reported on Google Code with ID 927

```
What steps will reproduce the problem?
1.  Upload a presentation
2.  Click and drag off screen
3.

What is the expected output? What do you see instead?

Expected that the presentation window will not allow presenter to drag a presentation
past its edges.  

For example, when a presentation first loads, the presenter should to be able to drag
it around.

When the presenter zooms in, the presenter should be able to drag it around until reaching
an edge.

Basically, add edge-detection to the draging of a presentation in the presenter's window.


Please use labels and text to provide additional information.

```

Reported by `ffdixon` on 2011-04-13 23:28:59
